dashboard/app/controllers/pd/workshop_enrollment_controller.rb

```diff
@@ -23,6 +23,44 @@ def new
         @enrollment.email = current_user.email
         @enrollment.email_confirmation = current_user.email
       end
+
+      session_dates = @workshop.sessions.map(&:formatted_date_with_start_and_end_times)
+
+      facilitators = @workshop.facilitators.map do |facilitator|
+        # TODO: Retire old K5 dashboard, and come up with more permanent solution that doesn't require cross-project file dependency.
```

Comment:
note the old K5 dashboard is retired and deleted now, so remove the first part of this TODO